packages/components/src/shareIcon/ShareIcon.js

```diff
@@ -4,16 +4,18 @@ import * as React from 'react';
 import { Platform } from 'react-native';
 import { Icon } from '@kiwicom/universal-components';
 
-export default function ShareIcon() {
+type Props = React.ElementProps<typeof Icon>;
```

Comment:
I am not sure about this type, since this gives no error: 

`<ShareIcon pippiLongStock="lol" />`

Reply:
Okay, I'll try and find a better one!